microcosm_flask/tests/conventions/test_query.py

```diff
@@ -0,0 +1,140 @@
+"""
+Query operation tests.
```

Comment:
Note that we'll often be able to extend the "search" operation for queries and should do that if possible. But if that really doesn't fit, we can do ad hoc queries.